modules/core/src/v2/wallet.ts

```diff
@@ -86,12 +93,13 @@ export interface PrebuildTransactionOptions {
 }
 
 export interface PrebuildAndSignTransactionOptions extends PrebuildTransactionOptions {
-    prebuildTx?: string;
+    prebuildTx?: string | PrebuildTransactionResult;
     verification?: VerificationOptions;
 }
 
 export interface PrebuildTransactionResult extends TransactionPrebuild {
     walletId: string;
+    consolidateId?: string;
```

Comment:
Since this is going in `wallet.js` -- one of the core modules in this library, you may want to leave an inline comment describing what a `consolidateId` is -- when it is used,etc. 

I can see someone looking at this code in 9 months and not completely understanding the nuance of this attr.

Reply:
that's fair, I've added some color